src/components/ItemEditorPage/RightPanel/RightPanel.tsx

```diff
@@ -109,36 +118,47 @@ export default class RightPanel extends React.PureComponent<Props, State> {
   }
 
   handleChangeCategory = (category: WearableCategory) => {
-    const data = {
+    let data: WearableData = {
       ...this.state.data!,
       category
     }
+    // when changing the category to SKIN we hide everything else
+    if (category === WearableCategory.SKIN) {
+      data = this.setReplaces(data, [])
+      data = this.setHides(data, [])
```

Comment:
If the category is sinks, shouldn't we be hiding the categories given by `getSkinHiddenCategories`?

Reply:
The categories that a skin hides are not embedded in the wearable, that logic lives in the explorer (all the skins hide the head/upper/lower/feet). What goes in the `hides` list is the extra stuff that this skin hides (ie. the accessories, like `hat`, `eyewear`, etc).